Gemfile

```diff
@@ -18,5 +18,7 @@ group :development do
   gem 'rubocop-rspec'
 end
 
+gem 'bolt', :git => 'https://github.com/barriserloth/bolt.git', :branch => 'pe30043/main/deprecate-harmful-bolt-server-terminology'
```

Comment:
I assume this should be taken out before merging?

Reply:
Ah yep, the commit message mentions it but I meant to leave a comment on this PR as well and I apparently didn't. That commit is there so that the `allowlist` spec change works, as an indication that this won't break specs in the actual codebase once the bolt-server allowlist PR is in builds